### PR TITLE
Ports CM defib sounds

### PR DESCRIPTION
**Pull request by Neocloudy**
## About The Pull Request
Ports CM's defibrillator sounds as the title says.
image
## Why It's Good For The Game
It's a *very* simple cosmetic change that in my experience makes the defib feel more responsive, and they're good sounds that add at least a bit of soul.
## Changelog
:cl:
add: Ports CM defib sounds. These play when toggling the defib, failing to revive someone or succeeding to revive someone.
/:cl:

### CODE REVIEW

**Comment by FosterGrime-Heymynameslouie** [comment]
very kewl

**Comment by Al-1ce** [comment]
Dare I say it, ..soul?

**Comment by Lumipharon** [comment]
Got a clip? Too lazy to download the files.

**Comment by Neocloudy** [comment]
> Got a clip? Too lazy to download the files.


https://user-images.githubusercontent.com/88008002/215474154-39df8b24-8750-4196-8ac8-0667cad21ad3.MOV

File: code/game/objects/items/defibrillator.dm [hunk 1]
```diff
@@ -103,6 +103,11 @@
 	user.visible_message(span_notice("[user] turns [src] [ready? "on and opens the cover" : "off and closes the cover"]."),
 	span_notice("You turn [src] [ready? "on and open the cover" : "off and close the cover"]."))
 	playsound(get_turf(src), "sparks", 25, TRUE, 4)
+	if(ready)
+		playsound(get_turf(src), 'sound/items/defib_safetyOn.ogg', 30, 0)
+	else
+		w_class = initial(w_class)
```
Comment (Lumipharon) [inline code/game/objects/items/defibrillator.dm:109]: is this just copy pasted from somewhere? This is not needed.

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Also the ready sound is unused?

File: code/game/objects/items/defibrillator.dm (hunk 1 above)
Comment (Neocloudy) [reply code/game/objects/items/defibrillator.dm:109]: deleting this and the ready sound very very soon

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**